Automattic/wp-calypso
# bring back highlighting to the plans SEO features when redirected from upgrade nudges

A step towards more controlled Plans page display ->  https://github.com/Automattic/wp-calypso/issues/17609. 
Plan feature highlighting when redirected is broken for the SEO upgrade Banner due to naming incompatibility between `selectedFeature` prop (Banner) and feature slug in `planProperties` (Plans component). 

**to test:**
- checkout this branch or use calypso.live
- go to `http://calypso.localhost:3000/settings/traffic/:jetpack_connected_site` 
- click on the `Search engine optimization` Banner
- verify that the `SEO Tools` feature is highlighted in the Pro plan section

- I looked for places the flag is used to assure other things won't break after the name change. IMO it is ok, but would be great to have another pair of eyes on that. 

![screen shot 2017-11-24 at 00 30 16](https://user-images.githubusercontent.com/13561163/33202691-c3e6f19c-d0fd-11e7-8e27-d9e4742cf414.png)
